src/pages/authpages/Login.css

```diff
@@ -0,0 +1,44 @@
+.main-container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 5rem;
+}
+
+.login-container {
+    width: 350px;
```

Comment:
Suggestion - You can use rem instead it will be useful in responsive layout also , instead of px for pixel value more than 10px